release-notes/9.0/linux-packages.json

```diff
@@ -0,0 +1,385 @@
+            "releases": [
+                {
+                    "name": "Alpine 3.20",
+                    "release": "3.20",
+                    "packages": [
+                        {
+                            "id": "libgcc",
+                            "name": "libgcc"
+                        },
+                        {
+                            "id": "ca-certificates",
+                            "name": "ca-certificates"
+                        },
+                        {
+                            "id": "libssl",
+                            "name": "libssl3"
+                        },
+                        {
+                            "id": "libstdc++",
+                            "name": "libstdc++"
+                        },
+                        {
+                            "id": "icu",
+                            "name": "icu-libs"
+                        },
+                        {
+                            "id": "icu",
+                            "name": "icu-data-full"
+                        },
+                        {
+                            "id": "tzdata",
+                            "name": "tzdata"
+                        },
+                        {
+                            "id": "krb5",
+                            "name": "libgssapi-krb5-2"
+                        }
+                    ]
+                },
+                {
+                    "name": "Alpine 3.19",
+                    "release": "3.19",
+                    "packages": [
+                        {
+                            "id": "libgcc",
+                            "name": "libgcc"
+                        },
+                        {
+                            "id": "ca-certificates",
+                            "name": "ca-certificates"
+                        },
+                        {
+                            "id": "libssl",
+                            "name": "libssl3"
+                        },
+                        {
+                            "id": "libstdc++",
+                            "name": "libstdc++"
+                        },
+                        {
+                            "id": "icu",
+                            "name": "icu-libs"
+                        },
+                        {
+                            "id": "icu",
+                            "name": "icu-data-full"
+                        },
+                        {
+                            "id": "tzdata",
+                            "name": "tzdata"
+                        },
+                        {
+                            "id": "krb5",
+                            "name": "libgssapi-krb5-2"
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "Debian",
+            "commands": [
+                "sudo apt-get update",
+                "sudo apt-get install -y --no-install-recommends"
```

Comment:
Good point. I agree that it would be more clear to have things separated out. I think something like the following should be sufficient:

```json
"commands": {
  "pre-install": "sudo apt-get update"
  "install": "sudo apt-get install -y --no-install-recommends"
}
```

Reply:
We solved a similar pre-install/install/query command problem in https://github.com/JL03-Yue/vscode-dotnet-runtime/blob/JL03-Yue/red-hat-sdk-install/vscode-dotnet-runtime-library/distro-data/distro-support.json. Could we re-use/adapt that?

Reply:
Yes. I think we could just take that as-is.

Reply:
How does this look?

https://github.com/dotnet/core/pull/9412/commits/d513a42d018ccd46ace0a72bcfd7e22d4029042d

I haven't updated the markdown generate code yet. Wanted to validate the schema change first.

Reply:
I updated the markdown generator to the new schema.

Reply:
That seems good. The file looks slightly different on our end now but I'm not sure you'd want to include the dpkg lock waits, it's a bit specific to our scenario. https://github.com/dotnet/vscode-dotnet-runtime/blob/main/vscode-dotnet-runtime-library/distro-data/distro-support.json

It's interesting that the documents for installing .NET on a specific distro are also quite tied to this. It would be good to keep these in parity. I don't know if that necessitates having a repo or some sort of service like the releases.json server to host it, but perhaps.

Reply:
Nope, no automation! But with the addition of these files, I'll be able to write automation for the docs which will help a lot.

Reply:
The supported-os.json file might also help.